rollup/rollupConfigBuilder.js

```diff
@@ -0,0 +1,39 @@
+const ui5 = require("./ui5-resolve");
+const builtins = require('builtin-modules');
+const { terser } = require("rollup-plugin-terser");
+const { nodeResolve } = require("@rollup/plugin-node-resolve");
+
+class ConfigBuilder {
+    static build(projectPath) {
+        return {
+            input: "./rollup/bundle-def.js",
+            plugins: [
+                ui5({
+                    assets: [
+                        "/resources/sap/ui/fl/**"
```

Comment:
here you maintain the assets, but in the rollup plugin it is maintained again

Reply:
Missed that part earlier, now it's used in plugin.

Reply:
👍